mk/uvmt/xrun.mk

```diff
@@ -85,18 +85,18 @@ endif
 # ADV_DEBUG=YES will enable Indago waves, default is to generate SimVision waves
 ifeq ($(call IS_YES,$(WAVES)),YES)
 ifeq ($(call IS_YES,$(ADV_DEBUG)),YES)
-XRUN_RUN_WAVES_FLAGS = -input ../../../tools/xrun/indago.tcl
+XRUN_RUN_WAVES_FLAGS = -input $(abspath $(MAKE_PATH)/../tools/xrun/indago.tcl)
 else
-XRUN_RUN_WAVES_FLAGS = -input ../../../tools/xrun/probe.tcl
+XRUN_RUN_WAVES_FLAGS = -input $(abspath $(MAKE_PATH)/../tools/xrun/probe.tcl)
 endif
 endif
 
 ################################################################################
 # Waveform (post-process) command line
 ifeq ($(call IS_YES,$(ADV_DEBUG)),YES)
-WAVES_CMD = cd $(XRUN_RESULTS)/$(TEST_NAME) && $(INDAGO) -db ida.db
+WAVES_CMD = cd $(XRUN_RESULTS)/$(CFG)/$(TEST_NAME) && $(INDAGO) -db ida.db
 else
-WAVES_CMD = cd $(XRUN_RESULTS)/$(TEST_NAME) && $(SIMVISION) waves.shm
+WAVES_CMD = cd $(XRUN_RESULTS)/$(CFG)/$(TEST_NAME) && $(SIMVISION) waves.shm
```

Comment:
With the run-index added, I _think_ this should now be:
`WAVES_CMD = cd $(XRUN_RESULTS)/$(CFG)/$(TEST_NAME)_$(RUN_INDEX) && $(SIMVISION) waves.shm`

Reply:
That sounds correct - some care needs to be taken to merge these makefile changes as several of them make changes to the same paths; this as a result of keeping the changes functional on their own and in separate pull-requests.  Feel free to ask if there are any further uncertainties.

Reply:
Can I ask you to make the changes?   You are more familiar with the original intent.

Reply:
I'm on it, might not get all the changes in before monday though.